splinterd/src/config/default.rs

```diff
@@ -0,0 +1,112 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::path::Path;
+
+use crate::config::{PartialConfig, PartialConfigBuilder};
+use crate::{
```

Comment:
Localize these here; it is the only place the defaults should exist after we are done with the config changes. (This probably means having them in both places temporarily.)

Reply:
Addressed. I re-imported them in main from the config module to avoid compiler warnings, which required further cfg attributes, but (as you said in your comment) this should be temporary.